docs/docs/50-running/55-memory.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Memory
+---
+You can configure Acorn apps to have a set memory upon startup.
+
+This allows you to specify memory that an Acorn will abide by when being created. There are a number of ways to set this so that the you have full control over defaults, indvidual Acorns, and maximums. In order of precedence, the ways to set memory for an Acorn are as follows:
+
+1. `acorn run --memory`
+2. Via the `memory` (or `mem`) property for workloads in the Acornfile.
+3. `acorn install --workload-memory-default`
+
+There is another install flag that this feature adds in the form of `--workload-memory-maximum`. This flag sets a maximum that when exceeded prevents the offending Acorn from being installed.
+
+:::info 
+Supported formats for memory flag values include
+- 1_234,       // 1234
+- 5M,          // 5_000_000
+- 1.5Gi,       // 1_610_612_736
+- 0x1000_0000, // 268_435_456
+
+These all translate into an exact amount of bytes. We encourage you use the binary representation of large quantities of bytes when interacting with memory such as Ki, Mi, Gi, and Pi.
+:::
+
+## Run
+Setting `memory` via `acorn run` has the highest order of precedence. When setting this, you operate under a new flag `--memory` (`-m` for short).
+
+### --memory
+When setting this value, you have two options - globally or per workload and you can do a combination of both. When setting the memory globally for that Acorn, you just define the memory you would like to set.
+
+```shell
+acorn run -m 512Mi foo
+```
+
+This will set all workloads in the `foo` acorn to have `512Mi` of memory. Adjacently, you can set the memory of each individual workload in the Acorn using a `workload=memory` pattern. 
+
+```shell
+acorn run -m nginx=512Mi foo
+```
+
+This will only update Acorn's `nginx` workload to have `512Mi` of memory.
+
+Finally, you can do a combination of both.
+
+```shell
+acorn run -m 256Mi,nginx=512Mi foo
+```
+
+This sets all workloads in the `foo` acorn to have `256Mi` of memory except for the `nginx` workload which will have `512Mi` of memory.
+
+
+## Acornfile
+Setting the memory via the Acornfile uses the `memory` property that is attached to all `workloads` (`containers` and `jobs`).
+
+```acorn
+containers: {
+    nginx: {
+        image: "nginx"
+        ports: publish: "80/http"
+        files: {
+            "/usr/share/nginx/html/index.html": "<h1>My first Acorn!</h1>"
+        }
+        memory: 512Mi
+    }
+}
+```
+
+> **Note**: This can be overwritten by `acorn run --memory`.
```

Comment:
use docusaurus admonitions https://docusaurus.io/docs/markdown-features/admonitions
```suggestion
:::note

This can be overwritten by `acorn run --memory`.

:::
```